### DISCUSSION TITLE
Globo News

### DISCUSSION THREAD

**Discussion by lgissoni** [iptv-org/community#707] [Q&A]
**Channel Name:** Globo News
**Channel ID:** GloboNews.br
**Country:** Brazil
**Language:** Portuguese
**Website:** https://g1.globo.com/globonews/
**Stream URL (optional):** xxx
**Notes (optional):** xxx

**Comment by ayushjain27** [comment]
Can you please tell me, in this we have to make page in which news of brazil  will be shown in portuguese language

**Comment by ayushjain27** [comment]
If this want to create please assign me.
/assign

**Comment by lgissoni** [comment]
Sorry I did not understand you.

**Comment by AntiPontifex** [comment]
Dead issue: please close.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19699
